spec/migrations/20170523220843_add_hawkular_id_to_miq_alerts.rb

```diff
@@ -0,0 +1,16 @@
+require_migration
+
+describe AddHawkularIdToMiqAlerts do
```

Comment:
@israel-hdez migrations have to go to the core app. Soon ™️ there will be a migrations repo for that